### PR TITLE
Add Pod Manager to index.html and alias

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

Added "Pod Manager" to explain a little better where the name "Podman" comes from.  Also added a new tag to show
you can alias podman for docker.

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
@rhatdan @mheon @vrothberg PTAL

File: index.html [hunk 1]
```diff
@@ -17,10 +17,12 @@
 </head>
 <h1>{{ page.title }}</h1>
 
-Welcome to the site for <a href="https://github.com/containers/libpod">Podman</a>. This site features announcements and news around Podman, and occasionally other <a href="https://github.com/containers/">container tooling</a> news.
+Welcome to the site for Pod Manager(<a href="https://github.com/containers/libpod">Podman</a>). This site features announcements and news around Podman, and occasionally other <a href="https://github.com/containers/">container tooling</a> news.
```
Comment (mheon) [inline index.html:20]: Maybe `<a href=...>Podman</a>, the pod manager`

Comment (vrothberg) [reply index.html:20]: I like that.

**Review by vrothberg — COMMENTED** [review]
Thank you so much, @TomSweeneyRedHat !

**Comment by rhatdan** [comment]
I merged this, but should we add a the 
Welcome to the site for `the` Pod Manager(Podman).

File: index.html (hunk 1 above)
Comment (TomSweeneyRedHat) [reply index.html:20]: @mheon I like the thought of having Podman out first in the line, but generally you have the full specification first, then the abbreviation in parens following.  Unless you've a strong opinion, I think I'll keep it as is, but will add 'the" in front of it.

File: index.html [hunk 2]
```diff
@@ -17,10 +17,12 @@
...
 
 <img src="images/podman.svg" alt="podman logo">
 
+<h3>What is Podman?  Simply put: `alias podman=docker`
```
Comment (gbraad) [inline index.html:24]: Shouldn't this have been `alias docker="podman"` instead?

so:
```
docker ps
```
would actually invoke:
```
podman ps
```

Comment (TomSweeneyRedHat) [reply index.html:24]: @gbraad Thanks for the issue!
Eack, my dyslexia struck again.  You're quite right.   I see you've added a PR, think I'll go merge that in.

**Pull request merged**